src/sentry/replays/lib/summarize.py

```diff
@@ -343,6 +343,8 @@ def as_log_message(event: dict[str, Any]) -> str | None:
                 return None
             case EventType.CLS:
                 return None
+            case EventType.NAVIGATION:
```

Comment:
We should add a comment explaining why we favor `NAVIGATION_SPAN` over this case